Assets/Mirror/Runtime/NetworkWriter.cs

```diff
@@ -12,11 +12,33 @@ public class NetworkWriter
 
         // create writer immediately with it's own buffer so no one can mess with it and so that we can resize it.
         // note: BinaryWriter allocates too much, so we only use a MemoryStream
-        readonly MemoryStream stream = new MemoryStream();
+        byte[] buffer = new byte[256];
```

Comment:
maybe start with 1400 bytes or even more?
1400 is MTU so on average this should be the max packet size where we never have to resize.
memory is pretty cheap. I'd argue it's fine to consume a bit more and never have to resize instead.

Reply:
To be exact, iirc 1492 is the normal MTU, so i'd rather start on 1500